nmrfx-utils/src/main/java/org/nmrfx/utils/properties/ListOperationItem.java

```diff
@@ -53,10 +53,10 @@ public class ListOperationItem extends OperationItem implements ObservableObject
      * @param description
      * @param typeSelector
      */
-    public ListOperationItem(ChangeListener<? super String> listener, List<String> defaultValue, String category, String name, String description, ChoiceOperationItem typeSelector) {
+    public ListOperationItem(ChangeListener<? super String> listener, List<?> defaultValue, String category, String name, String description, ChoiceOperationItem typeSelector) {
         super(category, name, description);
         if (defaultValue != null) {
-            this.defaultValue = (ArrayList<String>) defaultValue;
+            this.defaultValue = (ArrayList<?>) defaultValue;
```

Comment:
This is dangerous as well: someone could create a ListOperationItem with another List implementation, and that would crash with a ClassCastException. 

Several possibilties:
* change the field to List<?> if we can
* check the type with instanceof before cast
* ensure it's an arraylist by creating a new one before assignment

Of those, the best one would be to change the types from ArrayList to List but there are some impacts (getters should change as well, and their usage must be checked) and this is getting a bit far from the initial goal of this PR.

Reply:
Changed the value and defaultValue fields to List<?>